LM-1201/99-beta/hello-python.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""DevNet Express - Hello Python!
+
+Running this script will help you verify that your Python environment is setup
+correctly.  It will display information about your Python environment including
+the location of your Python interpreter, where your `pip install ...` packages
+are being installed and more.
+
+Usage:
+    python hello-python.py
+
+"""
+
+
+__author__ = "Chris Lunsford"
+__author_email__ = "chrlunsf@cisco.com"
+__copyright__ = "Copyright (c) 2018 Cisco and/or its affiliates."
+__license__ = "Cisco Sample Code License, Version 1.0"
+
+
+import os
+import sys
+
```

Comment:
Hm, I'm running this on python 2 and I guess I need 3? I'm getting:
```
AttributeError: 'module' object has no attribute 'implementation'
```

And, `pip install sys` says `Could not find a version that satisfies the requirement sys (from versions: )
No matching distribution found for sys` so I think I need to double-check version requirements with you. I'm installing python3 now with Homebrew on Mac and I'll let you know how it goes... sending review now since I know you want to turn this around quickly.